source/extension-manifest.json

```diff
@@ -207,13 +207,23 @@
                         "resultSelector": "jsonpath:$.[*]"
                     },
                     {
-                        "name": "OctopusListProjects",
-                        "endpointUrl": "$(endpoint.url)/api/projects?partialName=$(PartialNameFilter)",
+                        "name": "OctopusAllProjectGroups",
+                        "endpointUrl": "$(endpoint.url)/api/projectgroups/all",
+                        "resultSelector": "jsonpath:$.[*]"
+                    },
+                    {
+                        "name": "OctopusAllProjectGroupsInSpace",
+                        "endpointUrl": "$(endpoint.url)/api/$(SpaceId)/projectgroups/all",
+                        "resultSelector": "jsonpath:$.[*]"
+                    },
+                    {
```

Comment:
We now grab the top 500 projects within this project group. If you have more than that, or hit the 2MB response limit, tough cookies. You need to split up your project groups to manage less projects. This is the workaround customers at scale can use to get around the response limits.